### ISSUE TITLE
Verbose Error Flag Request

### ISSUE THREAD

**Issue by VinLucero** [moov-io/metro2#37]
Metro2 Version: `v0.2.6`

**What were you trying to do?**
Print more verbose errors on what field / location in the input file is causing the error.

**What did you expect to see?**
A specific part of the file which caused the program to error so I could resolve it.

**What did you see?**
Generic error

![image](https://user-images.githubusercontent.com/3068574/98280802-c8b3bf80-1f50-11eb-9385-a7667ade3c20.png)

**How can we reproduce the problem?**

Run the print command on unpacked_fixed_file.dat for instance.

**Comment by adamdecaf** [comment]
Agreed. I think these enhanced messages should be the default. Seems we were just naive before when returning errors.

**Comment by mfdeveloper508** [comment]
I am tried to print error message contained exact field name and record(segment) name and reason

example:

./metro2 validator --input test/testdata/unpacked_fixed_file.dat

Error: total_consumer_segments_j1 in trailer record has an invalid value

**Comment by mfdeveloper508** [comment]
please let me know another approach to have nice error message

**Comment by mattbernardo** [comment]
@adamdecaf Will @mfdeveloper508's updates get pushed to a docker hub build?

**Comment by adamdecaf** [comment]
@mattbernardo yea we can make a v0.3.0 release. 

@mfdeveloper508 do you have anything else to add before a release is made?

**Comment by mfdeveloper508** [comment]
I haven't now

**Comment by adamdecaf** [comment]
@mattbernardo v0.3.1 is released, can you try it out? Let us know if you have any problems or questions. Thanks!
